src/DynamoManipulation/NodeManipulator.cs

```diff
@@ -498,15 +498,24 @@ private RenderPackageCache GenerateRenderPackages()
             // and we must return immediately before proceeding with further calls to ProtoGeometry
             if (IsNodeNull(Node.CachedValue)) return packages;
 
-            AssignInputNodes();
-            
-            active = UpdatePosition();
-
-            if (!IsEnabled())
+            //it's possible the the node's return value is not null, but libG was not loaded correclty
+            //it could be a function, null check is not sufficent, wrap use of libG in a try catch.
```

Comment:
Instead of a try-catch, can we add a check for the node being a function inside `IsEnabled `and return false if that's the case?

Reply:
`NodeModel.CachedValue.IsFunction`

Reply:
no, I tried that already, it appears the direct manipulator should be created when the node returns a function so if we do that, direct manipulation will not work at all.

Reply:
I guess my comment is not sufficient if it did not explain the issue, I will try to add more detail there,

Reply:
Direct manipulation is supposed to work for a node that is not connected if it's input port has a default arg or if it's connected to a slider. I'm not sure if it's supposed to work if it's a function.